bin/node-sass

```diff
@@ -148,25 +148,30 @@ function getOptions(args, options) {
  */
 
 function watch(options, emitter) {
-  var dir = options.watch;
-  var gaze = new Gaze();
+  var watchDir = isSassFile(options.watch) ? path.dirname(path.resolve(options.watch)) : options.watch,
+      glob = options.recursive ? '**/*.{sass,scss}' : '*.{sass,scss}',
+      gaze = new Gaze(),
+      dir;
```

Comment:
I had that, but the top of the file was using commas, so I changed it.  Are there contribution docs on this stuff that I can read?